### ISSUE TITLE
Only X/Y overrides  Z behaviour on Custom support

### ISSUE THREAD

**Issue by 5axes** [Ultimaker/Cura#8082]
**Application version**
4.6.1  

When we use a custom support the result is always generate as if the  X/Y overrides  Z options is used even if in the settings we have activated the Z overrides X/Y  in the Support Distance Priority. on the joined image and project , you can see that on the left where we have a custom support the slicing result for the support is different from the regular support.

![Support_Perso](https://user-images.githubusercontent.com/11015345/87851644-fe2d9200-c8fa-11ea-9808-f2088f791087.JPG)

Hereafter a sample project
[testsup.zip](https://github.com/Ultimaker/Cura/files/4941566/testsup.zip)

**Comment by Ghostkeeper** [comment]
Indeed, this is easy to reproduce. Your project file shows it very well. We're adding this to our backlog (CURA-7610 for other developers).
